teacher.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization)
+    super(37, 'Abiyu')
+    @specialization = specialization
+  end
```

Comment:
- [ ] Firstly, the requirement does not state setting a `getter and setter` for specialization, so kindly commit that 👍 
- [ ] Secondly, just as I stated in the student class, also include the parameters of the person class in the `super` call instead of inputting fixed data.

### Requirement:
![image](https://github.com/binyamolango/OOP-school-library/assets/50990925/19f431f7-6625-47c0-9888-8299267f6581)

Reply:
well resolved! ✔️